packages/sdk-react/src/server.ts

```diff
@@ -0,0 +1,25 @@
+export {
+  GrowthBook,
+  setPolyfills,
+  configureCache,
+  clearCache,
+} from "@growthbook/growthbook";
+
+export type {
+  Context,
+  Experiment,
+  Result,
+  ExperimentOverride,
+  Attributes,
+  ConditionInterface,
+  ExperimentStatus,
+  FeatureDefinition,
+  FeatureResult,
+  FeatureResultSource,
+  FeatureRule,
+  JSONValue,
+  SubscriptionFunction,
+  Filter,
+  VariationMeta,
+  VariationRange,
+} from "@growthbook/growthbook";
```

Comment:
Is there a reason we can't just do something like
```
export  * as server from "@growthbook/growthbook";
```

Reply:
I'm not sure why we'd want to do that. Could you explain the potential benefit?

The reason I did it this way was to mimic how we currently do it for the React SDK today, just with React client-side methods omitted.